src/category.rs

```diff
@@ -1,12 +1,11 @@
 use std::path::PathBuf;
 
-use rocket::http::RawStr;
 use rocket::request::FromParam;
 
 fn is_category(name: &str) -> bool {
     let mut path = PathBuf::from("templates");
     path.push(name);
-    path.push("index.hbs");
+    path.push("index.html.hbs");
```

Comment:
`rocket_dyn_templates` (previously part of `rocket_contrib`) [sets content type based on the second file extension](https://docs.rs/rocket_dyn_templates/0.1.0-rc.3/rocket_dyn_templates/#content-type). So I renamed all the templates, otherwise the html would be sent as `text/plain`.